### DISCUSSION TITLE
Is Tailwind required?

### DISCUSSION THREAD

**Discussion by samal-rasmussen** [themesberg/flowbite-svelte#1109] [Q&A] [answered]
I am not emotionally ready to learn Tailwind. I know everyone and their grandmas have adopted it - I just ain't ready, I'm too emotionally drained by everything else as is. When I open the Tailwind docs I just feel drained.

Flowbite Svelte looks great, but the docs are all with Tailwind. Is Tailwind required to use Flowbite Svelte?

**Comment by shinokada** [comment, marked as answer]
Yes, you need to install Tailwind. 
https://flowbite-svelte.com/docs/pages/quickstart
